core/src/logging.rs

```diff
@@ -62,7 +55,26 @@ fn do_init() {
         registry.with(layer.with_tracer(tracer).with_filter(filter))
     };
 
-    registry.init();
+    let stdout_layer =
+        tracing_subscriber::fmt::layer().with_filter(LevelFilter::from(stdout_level));
+
+    let registry = tracing_subscriber::registry().with(stdout_layer);
+
+    let maybe_file_layer = if let Some((level, output_dir)) = file_level {
+        message += &format!(", file (in dir {output_dir:?})");
+
+        let file_appender = RollingFileAppender::new(Rotation::DAILY, output_dir, "sk.log");
```

Comment:
What is the `output_dir` by default? (It's not immediately apparent to me from the code)